src/main/java/com/agata/jeeshop/controllers/CartViewController.java

```diff
@@ -27,9 +28,9 @@ public ModelAndView getCartView(@PathVariable("id") final long id){
         return mav;
     }
 
-   /*   @DeleteMapping("/{id}/{itemId}")
-        public ResponseEntity<ProductDto> removeProduct(@PathVariable("id") final long id, @PathVariable("itemId") final long itemId){
-         cartService.removeProduct(id,itemId);
-       return new ResponseEntity<>(HttpStatus.OK);
-      }*/
+     @DeleteMapping("/{cartId}/{cartItemId}")
```

Comment:
Tutaj powiedziałem o innym rozwiązaniu, ale to zastosowane jest poprawne (moje było złe).